api/src/main/java/com/github/streamshub/console/api/support/OASModelFilter.java

```diff
@@ -117,12 +119,31 @@ public void filterOpenAPI(OpenAPI openAPI) {
         openAPI.getComponents().setSchemas(new TreeMap<>(openAPI.getComponents().getSchemas()));
         // Prune any schemas no longer referenced
         FilterUtil.applyFilter(new UnusedSchemaFilter(), openAPI);
+        Optional<ConsoleConfig> consoleConfig = Optional.empty();
 
-        config.getOptionalValue("quarkus.oidc.auth-server-url", String.class)
-            .ifPresent(url -> Optional.of(openAPI.getComponents())
-                    .map(Components::getSecuritySchemes)
-                    .map(schemes -> schemes.get("ConsoleSecurity"))
-                    .ifPresent(scheme -> scheme.setOpenIdConnectUrl(url)));
+        try {
+            Instance<ConsoleConfig> configInstance = CDI.current().select(ConsoleConfig.class);
+
+            if (configInstance.isResolvable()) {
+                consoleConfig = Optional.of(configInstance.get());
+            }
+        } catch (Exception e) {
+            // Ignore, expected at build time where CDI may not be available
```

Comment:
Should we at least log it?

Reply:
I looked a bit more closely and it turns out the filter will _not_ run at build time anyway, so I will refactor this slightly. The exception handling doesn't need to be split because of that.